index.js

```diff
@@ -7,7 +7,7 @@ const npmFetch = require('npm-registry-fetch')
 const SearchOpts = figgyPudding({
   detailed: { default: false },
   limit: { default: 20 },
-  from: { default: 0 },
+  offset: { default: 0 },
```

Comment:
I understand that `offset` is a commonly-used name for this, but I'd rather not arbitrarily rename the API's parameter. Is there a particular reason you're changing this? Keep in mind that this would make this PR a semver-breaking change.

Reply:
Beacause I have seen in the documentation that you have used `offset` instead of `from`:

```markdown
opts.offset - Offset number for results. Used with opts.limit for pagination. Default: 0
````

Reply:
oh noooo. That's totally my bad. Thanks for spotting that. I think we should -definitely- be following [what the registry docs say](https://github.com/npm/registry/blob/master/docs/REGISTRY-API.md#get-v1search). Do you mind editing the docs in this PR as well? Everything lgtm besides that and I'll run a release for you once it's merged. 👍

Reply:
Yes of course, I will change it right now 👍

Reply:
Done 👍